packages/turf-boolean-point-on-line/index.ts

```diff
@@ -82,7 +87,11 @@ function isPointOnLineSegment(
   const dyc = pt[1] - y1;
   const dxl = x2 - x1;
   const dyl = y2 - y1;
-  const cross = dxc * dyl - dyc * dxl;
+  let cross = dxc * dyl - dyc * dxl;
+  if (precision) {
```

Comment:
I think this is more commonly expressed as `epsilon`. You could rename the parameter and default it to `1e-10`.
https://stackoverflow.com/a/328122

Reply:
In terms of renaming the parameter to epsilon, do you have a problem with keeping public facing parameter (the line you commented on above) as `precision` or should that be named `epsilon` as well? It seems more consumer friendly as precision but I'd like to keep the naming conventions consistent within this domain that I have little experience in.